src/redux/books/Book.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Book({ author, title }) {
+  return (
+    <div>
+      <h1 className="bookTitle">{title}</h1>
+      <p className="bookAuthor">{author}</p>
+    </div>
+  );
+}
+
+export default Book;
```

Comment:
- [OPTIONAL] I think it would be better to put `Book.js` in the `components` directory instead of `redux` folder, because this file contains normal/basic React code (there is no redux here). The same think with `DisplayBooks.js`, `Form.js` and `NavBar.js` ( these files are components ) 💯